Automattic/newspack-blocks
# feat(ras-acc): update translation files

### All Submissions:






### Changes proposed in this Pull Request:

This PR updates the translation files for the Newspack Blocks plugin for RAS-ACC.

We may need to do this again after the beta testing bugs are fixed, but this should at least capture the bulk of the changes that have been made as part of this project.

See: 1205234045751551-as-1206125636101544

### How to test the changes in this Pull Request:

1. Review the updated .POT file and confirm that things look okay (it should basically have a bunch of strings added for RAS-ACC).
2. Review the .po files included in the plugin to confirm no existing translations have been removed by accident.
3. Apply the PR.
4. Navigate to WP Admin > General, and switch your site's language to one of the ones we have translations for, like Portuguese (make sure it's one that actually includes translations for the block you test in the next step).
5. Add a block to the editor; a good bet is the Donate block. Confirm the labels are translated.
6. Publish and view on the front-end; confirm that translations are being loaded there, too.

### Other information:
